vkubelet/lookup_windows.go

```
@@ -43,7 +44,8 @@ func lookupProvider(provider, providerConfig string, rm *manager.ResourceManager
 	case "sfmesh":
 		return sfmesh.NewSFMeshProvider(rm, nodeName, operatingSystem, internalIP, daemonEndpointPort)
 	default:
-		fmt.Printf("Provider '%s' is not supported\n", provider)
+		// TODO(@cpuguy83): perahps this should return an error

```

typo: perhaps
let's return an error.